README.md

```diff
@@ -93,11 +93,17 @@ The password and cryptographic keys are created when registering and stored in t
 ## Building
 
 This project uses [Gradle](http://gradle.org) for building and maintaining dependencies. If you have a recent gradle
-version installed, you can replace `./gradlew` with `gradle` in the following steps.
+version installed, you can replace `./gradlew` with `gradle` in the following steps. Alternatively, you can install the prerequisites with:
+```
+yes | sudo apt-get install default-jre
+yes | sudo apt-get install openjdk-17-jdk
+export JAVA_HOME=/usr/lib/jvm/java-1.17.0-openjdk-amd64
```

Comment:
The java installation is OS dependent. I think it would be better to create a wiki page for more detailed build instructions and just add a link to that in the README.